examples/OneOpenAir/OneOpenAir.ino

```diff
@@ -857,11 +883,12 @@ static void configUpdateHandle() {
   fwNewVersion = configuration.newFirmwareVersion();
   if (fwNewVersion.length()) {
     bool doOta = false;
-    if (measurements.otaBootCount == 0) {
+    if (measurements.otaBootCount < 0) {
       doOta = true;
       Serial.println("First OTA");
```

Comment:
I think the message is misleading, the device might have gone through several OTA updates when we arrive here.

Reply:
It's indicate first OTA check. If it's zero and process failed, retry will call for 3-4 times (Because we check cloud configure each 15sec). < 0 should help, that mean the otaBootCount is invalid, after first check it's set to bootCount